src/main/java/com/epam/aidial/core/util/BufferingReadStream.java

```diff
@@ -134,38 +141,37 @@ private synchronized void handleChunk(Buffer chunk) {
             // stop streaming
             return;
         }
-        if (isStreaming && isLastChunk(content)) {
-            lastChunkPos = pos;
+        if (eventStreamParser != null) {
```

Comment:
require some comments, hard to understand what's going on

Reply:
fixed